src/graphql/schema.gql

```diff
@@ -402,8 +402,8 @@ type SpaceFilters {
 }
 
 type SpaceVoting {
-  delay: Int
-  period: Int
+  delay: Float
+  period: Float
```

Comment:
`Float` can allow floating point values, don't think it is ideal here

Reply:
Found this 
```javascript
const { GraphQLScalarType } = require('graphql');

const LargeInt = new GraphQLScalarType({
  name: 'LargeInt',
  description: 'Large integer type',
  serialize(value) {
    return value.toString(); // Convert outgoing large integer to string
  },
  parseValue(value) {
    return BigInt(value); // Convert incoming string to large integer
  },
  parseLiteral(ast) {
    if (ast.kind === Kind.STRING) {
      return BigInt(ast.value); // Convert string from query to large integer
    }
    return null;
  },
});
```

Reply:
> `Float` can allow floating point values, don't think it is ideal here

It can, but as we have already check in place to ensure these 2 are always integer, graphql will always show these as int, even with the `Float` type.

We can add new check to prevent new large values, but in my mind, editing existing space manually to fix wrong values was never an option.

Reply:
> It can, but as we have already check in place to ensure these 2 are always integer, graphql will always show these as int, even with the Float type.

Yes, but `Float` type may confuse graphql API users, where they expect it to float but we always send INT

Reply:
> We can add new check to prevent new large values, but in my mind, editing existing space manually to fix wrong values was never an option.

Yea bad option 🙈 but not sure if there is some alternative way